.eslintrc.js

```diff
@@ -20,7 +20,6 @@ module.exports = {
     // node files
     {
       files: [
-        '.template-lintrc.js',
```

Comment:
I removed this because I was getting ridiculous HTML indentation errors - are you ok with that?